src/interactive/setup.py

```diff
@@ -5,10 +5,17 @@
 # --------------------------------------------------------------------------------------------
 
 from codecs import open
+import os
+import re
 from setuptools import setup, find_packages
 
-# Version is also defined in azclishell.__init__.py.
-VERSION = "0.4.1"
+# Version is defined in azclishell.__init__.py.
```

Comment:
Is it possible for interactive to get at the version defined here in setup.py through code, instead of the other way around? Maybe similarly to how in the core CLI we can get the version of every module.